_posts/2020-04-29-newpipe-0.19.3-release.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: post
+title: "NewPipe 0.19.3 released: Android TV, YouTube music, greeting new team members and so much more"
+short: "NewPipe 0.19.3 released"
+date: 2020-04-29 00:00:00 +0100
+categories: [release, pinned]
+author: Poolitzer
+image: newpipe
+excerpt_separator: <!-- more -->
+---
+
+Another day in quarantine, another release of NewPipe. This one comes with a lot of bug fixes, improvements, more features and more NewPipeTeam members. It can not get better then that.
+
+<!-- more -->
+
+## Android TV
+
+"Hey, release-post guy" I hear you say, "I can already run NewPipe on my Android TV, what is the difference now?" Well now, you can actually use it. You can scroll long video descriptions, can focus what element you want to focus on (and have the first one focused whenever you open a page which is important. I think.), the native keyboard is used instead of an on-screen one, you can seek videos however you like, no annoying ripple effects -  it is great. So great, you should probably buy an Android TV just for using NewPipe! All of this happened mostly thanks to [@Alexander--](https://github.com/Alexander--). They made their first contribution back in October, but now found time to come back and enhanced the TV experience greatly, thank you. Other people involved in this are [@B0pol](https://github.com/B0pol) (who improved the detection of non standard devices as Android TVs), [@revathskumar](https://github.com/revathskumar) (who found issues), [@Pentaphon](https://github.com/Pentaphon), [@christianfl](https://github.com/christianfl), [@Disonantemus](https://github.com/Disonantemus) (all found an issue), [@sorryusernameisalreadytaken](https://github.com/sorryusernameisalreadytaken) (who found a lot of issues) and [@TobiGr](https://github.com/TobiGr) (who was himself).
```

Comment:
guy,"
Well, now
scroll through (not sure if that's the best preposition there)
important I think), or maybe important, I think),
effects - it (i.e. no double space)
I was also involved in it: I made it possible to open the channel by long clicking a comment.